Integrations/integration-CounterTack.yml

```diff
@@ -0,0 +1,1716 @@
+            'command name': response.get('command_name'),
+            'status': response.get('status'),
+        }
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': quarantine_context,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('The command has been applied successfully:', data),
+            'EntryContext': context
+        }
+        demisto.results(entry)
+
+    def disable_quarantine():
+
+        """
+        Allows a previously quarantined endpoint to communicate with the network.
+
+        demisto parameter: (string) endpoint_id
+            The unique ID of the endpoint
+        demisto parameter: (string) type
+            The type of the command: lift_quarantine
+
+        """
+
+        endpoint_id = demisto.args().get('endpoint_id')
+        body = demisto.args().get('type')
+        body = {
+            'type': body
+        }
+        response = endpoint_quarantine_request(endpoint_id,body)
+        quarantine_response = get_endpoint_request(endpoint_id)
+
+        quarantine_context = {
+            'endpoint_ID': endpoint_id,
+            'is_quarantine': quarantine_response.get('is_quarantined')
+        }
+
+        context['CounterTack.Endpoint(val.ID && val.ID === obj.ID)'] = quarantine_context
+
+        data = {
+            'ID': response.get('id'),
+            'user name': response.get('username'),
+            'request time': response.get('request_time'),
+            'endpoint ID': response.get('endpoint_ids'),
+            'command name': response.get('command_name'),
+            'status': response.get('status'),
+        }
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': quarantine_context,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('The command has been applied successfully:', data),
+            'EntryContext': context
+        }
+        demisto.results(entry)
+
+    def file_extract_request(endpoint_id,body):
+
+        """
+        Request for extracting file from specified endpoint
+        """
+
+        url = '{}/endpoints/{}/commands'.format(BASE_PATH,endpoint_id)
+
+        response = http_request(
+            'POST',
+            url,
+            headers = GET_HEADERS,
+            body = json.dumps(body)
+        )
+        return response
+
+    def extract_file():
+
+        """
+        Enables an API consumer to extract the file in addition to some file metadata.
+
+        demisto parameter: (string) endpoint_id
+            The unique ID of the endpoint
+        demisto parameter: (string) body
+            The type of the command: extract file and the file path
+        """
+
+        endpoint_id = demisto.args().get('endpoint_id')
+
+        args = demisto.args()
```

Comment:
why are you doing all of a sudden? need to be consistent in all functions, so please remove this